### ISSUE TITLE
Code export throws syntax error

### ISSUE THREAD

**Issue by nielsrolf** [man-group/dtale#111]
Hi!
Executing the exported code is currently throwing a syntax error for me:

```Python
if isinstance(df, (pd.DatetimeIndex, pd.MultiIndex)):
    df = df.to_frame(index=False)

# remove any pre-existing indices for ease of use in the D-Tale code, but this is not required
df = df.reset_index().drop('index', axis=1, errors='ignore')
df.columns = [str(c) for c in df.columns]  # update columns to strings in case they are numbers

df = df.query('')

chart_data = pd.concat([
    df['date'],
    df['RPP'],
], axis=1)
chart_data = chart_data.sort_values(['date'])
chart_data.columns = ['x', 'RPP']
chart_data = chart_data.groupby('x')[['['RPP']']].sum().reset_index()
chart_data = chart_data.dropna()
```

Output:
```
  File "<ipython-input-20-c9a852059ab4>", line 16
    chart_data = chart_data.groupby('x')[['['RPP']']].sum().reset_index()
                                               ^
SyntaxError: invalid syntax
```

**Comment by aschonfeld** [comment]
@nielsrolf good catch!  I'll update accordingly.

**Comment by aschonfeld** [comment]
fixed in v1.8.1
